docs/section-2-using-parsers.md

```diff
@@ -615,6 +615,28 @@ For example, this pattern would match any node inside a call:
 (call (_) @call.inner)
 ```
 
+#### Special Nodes
+
+When the parser encounters text it does not recognize, it represents this node
+as `(ERROR)` in the syntax tree. These error nodes can be queried just like
+normal nodes:
+
+```scheme
+(ERROR) @the-error
+```
+
+Similarly, if a parser is almost able to parse a document but it detects a
+missing node, it will insert that missing node into the tree to preserve the
+rest of the document. These missing nodes appear as regular nodes in the tree
+(but with zero width), and can be queried using the `(MISSING)` alias:
```

Comment:
Adjust the copy

```suggestion
Similarly, if a parser is able to recover from erroneous text by inserting a missing token and then reducing, it will insert that missing node in the final tree so long as that tree has the lowest error cost. These missing nodes appear as seemingly normal nodes in the tree, but they are zero tokens wide, and are a property of the actual terminal node that was inserted, instead of being its own kind of node. These special missing nodes can be queried using `(MISSING)`:
```